internal/telemetry/reporter_test.go

```diff
@@ -58,14 +58,14 @@ func TestWriteTimeSeries(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			var handlerWriteErr error
```

Comment:
This is similar to the no-require-in-goroutines lint fix, since http handlers are run inside of goroutines.